### PR TITLE
Feature: Add dummy server to test sniffing

### PR DESCRIPTION
**Pull request by shravan20**
Description:

To start a dummy server along with application bootup. This is for better devEx

### CODE REVIEW

**Comment by idodav** [comment]
@shravan20 When creating a pull request. Please create a new branch and link it to the issue. 
Also use the "Feature:" prefix for the pull request.

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "build:sniffer": "cd ./traffic-sniffer && npm run build",
     "build": "concurrently --kill-others \"cd ./traffic-sniffer && npm run dev\"  \"cd ./traffic-dashboard && npm run build\"",
     "dev": "concurrently --kill-others \"cd ./traffic-sniffer && npm run dev\"  \"cd ./traffic-dashboard && npm run dev\"",
+    "dev-server": "npm run dev && nodemon dummy-server.js && npm run dev",
```
Comment (idodav) [inline package.json:18]: Remove the second npm run dev.
And consider using concurrently.

**Review by idodav — CHANGES_REQUESTED** [review]

File: dummy-server.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
+
+/**
+ * This is just a dummy `test-server` to ensure sniffing is working properly
+ */
+app.get("/", (_req, res) => {
```
Comment (idodav) [inline dummy-server.js:7]: maybe use "*".
So all the requests will return a response.

File: package.json (hunk 1 above)
Comment (idodav) [reply package.json:18]: consider changing the command to demo?

File: dummy-server.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const express = require('express');
+const app = express();
```
Comment (idodav) [inline dummy-server.js:2]: Consider adding a setInterval that sends requests to the sharkio dashboard.

File: package.json (hunk 1 above)
Comment (shravan20) [reply package.json]: Resolved

**Review by idodav — APPROVED** [review]

**Pull request merged**